crates/fluvio-cli/src/topic/create.rs

```diff
@@ -121,10 +121,15 @@ impl CreateTopicOpt {
             })
         };
 
-        let is_valid = hostname_validator::is_valid(&self.topic);
+        let is_valid = self
+            .topic
```

Comment:
Not sure where I should put a function with this validation logic, is it ok in fluvio-sc-schema? @sehz